sites/pycharm-guide/contents/tutorials/intro-aws/setup/index.md

```diff
@@ -0,0 +1,114 @@
+
+
+# AWS SAM (Serverless Application Model)
+
+![AWS SAM](./aws_sam.png)
+
+According to Amazon, The AWS Serverless Application Model (SAM) is an open-source framework for building serverless applications.
+A serverless application is a combination of Lambda functions, 
+event sources, and other resources that work together to perform tasks.
+Note that a serverless application is more than just a 
+Lambda function—it can include additional resources such as APIs,
+databases, and event source mappings.
+
+You can download the setup file from `aws.amazon.com/serverless/sam` and follow the 
+same installation process like you did above for the CLI.
+
+
+# AWS Toolkit
+
+The AWS Toolkit for PyCharm is an open source plug-in for the PyCharm IDE
+that makes it easier to create, debug, and deploy Python applications on
+Amazon Web Services. The toolkit provides an integrated experience for 
+developing serverless applications, including assistance for getting 
+started, step-through debugging, and deploying from the IDE.
+
+If you are interested to know more about the plugin
+then refer this [link](https://aws.amazon.com/pycharm/).
+
+To install the plugin follow the below steps.
+
+Go to `Settings -> Plugins` search `AWS Toolkit` and click on `Install`.
+
+
+![AWS Toolkit](./aws_toolkit.png)
+
+
+# AWS Management Console
+
+Before starting up the project make sure that you have an AWS Account. 
+As you may know, AWS provides a free tier for some of the services,
+up to 12 months. Also, there are some services which are even free 
+after 12 months, such as AWS Lambda, which provides 1 million free
+requests per month and 400,000 GB-seconds of compute time per month.
+
+To know more about AWS, please visit [aws.amazon.com](https://aws.amazon.com/)
+
+![AWS Management Console](./aws_management_console.png)
+
+
+# Identity & Access Management (IAM)
+
+Once you have created an account and successfully logged into AWS, then we 
+will goto IAM section. IAM stands for Identity & Access Management. IAM lets you manage access to AWS services and resources securely. 
+
+![IAM](./iam.png)
+
+
+When I click on users, basically it will show me the lists of users 
+residing in my account. As you can see, I have one user. I will click
+on that specific user to get more information.
+
+![IAM_USER](./iam_user.png)
+
+You can see below that user falls under a <strong>Group</strong>. 
+A group is a collection of IAM users. Groups let you specify permissions for multiple users and make it easier to manage.
+
+![IAM_USER_GROUP](./aws_iam_group.png)
+
+Over here I have provided full access to some of the AWS Services 
+because of the tutorial, but this definitely is not a good practice.
+You should always follow the concept of least privilege. 
+Okay, we are done with the Groups. Let’s move back to the Users. 
+
+I will click on the specific user. Now I will hover to <strong>Security Credentials.</strong>
+I am now going to click on create access key and it is 
```

Comment:
When we tell people something in the UI to click, let's make it plaintext, such as: click on `Create Access Key`